LICENSE.txt

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 SendGrid, Inc.
+Copyright (c) 2017 SendGrid, Inc.
```

Comment:
Can you please update the year to be a range? This is my fault for assuming that would already have been there... Should be: 
```
Copyright (c) 2016-2017 SendGrid, Inc.
```

Reply:
Fixed